config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_dnsrecordsets.dns.cnrm.cloud.google.com.yaml

```diff
@@ -157,7 +157,7 @@ spec:
                                       value is case-sensitive. Possible values: ["regionalL4ilb",
                                       "regionalL7ilb", "globalL7ilb"].'
                                     type: string
-                                  networkUrlRef:
+                                  networkRef:
```

Comment:
I recall that we agreed to keep the name `networkUrlRef`? Or did I miss anything? @jingyih

Reply:
@jingyih found out yesterday, that we have been using networkRef instead of networkUrlRef for other within DNS. I made this change to keep it consistent within DNS.

Reply:
I believe consistency within the API is not a goal and is not our guideline for reference naming. Details can be found at https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1399#discussion_r1558344708. We probably need to follow up offline to agree on a solution.

Reply:
What @jingyih found is that we already started renaming networkURLRef -> networkRef in DNS: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/204782804df09f0ec10187bcdc7b13891b0203b7/config/servicemappings/dns.yaml#L45-L48

To me, that's pretty compelling.  And I had already merged the networkUrlRef PR, and then we found the counter-evidence, and rather than revert, @anhdle-sso agreed to fix it up (which is this PR)